source/_components/zha.markdown

```diff
@@ -63,7 +63,19 @@ The custom quirks implementations for zigpy implemented as ZHA Device Handlers f
   - [ConBee USB adapter from Dresden-Elektronik](https://www.dresden-elektronik.de/conbee/)
   - [RaspBee Raspberry Pi Shield from Dresden-Elektronik](https://www.dresden-elektronik.de/raspbee/)
 
-## Configuration
+## Configuration - GUI
+Visit `http://homeassistant-ip:8123/config/integrations/dashboard` which will put you on the `configuration` -> `integrations` page.
```

Comment:
Please avoid this fake URL. Instead, describe it. For example: From the Home Assistant frontend to the the **Configuration** menu and click on **Integrations**. Or similar.
Please avoid the use of backticks here, since it is not describing any code like.